packages/create-turbo/src/commands/create/createProject.ts

```diff
@@ -0,0 +1,231 @@
+    );
+    process.exit(1);
+  }
+
+  const originalDirectory = process.cwd();
+  process.chdir(root);
+
+  /**
+   * clone the example repository
+   */
+  const loader = turboLoader("Downloading files...");
+  try {
+    if (repoInfo) {
+      const repoInfo2 = repoInfo;
+      console.log(
+        `\nDownloading files from repo ${chalk.cyan(
+          example
+        )}. This might take a moment.`
+      );
+      console.log();
+      loader.start();
+      await retry(() => downloadAndExtractRepo(root, repoInfo2), {
+        retries: 3,
+      });
+    } else {
+      console.log(
+        `\nDownloading files${
+          !isDefaultExample ? ` for example ${chalk.cyan(example)}` : ""
+        }. This might take a moment.`
+      );
+      console.log();
+      loader.start();
+      await retry(() => downloadAndExtractExample(root, example), {
+        retries: 3,
+      });
+    }
+  } catch (reason) {
+    function isErrorLike(err: unknown): err is { message: string } {
+      return (
+        typeof err === "object" &&
+        err !== null &&
+        typeof (err as { message?: unknown }).message === "string"
+      );
+    }
+    throw new DownloadError(isErrorLike(reason) ? reason.message : reason + "");
+  } finally {
+    loader.stop();
+  }
+
+  const rootPackageJsonPath = path.join(root, "package.json");
+  const rootMetaJsonPath = path.join(root, "meta.json");
+  const ignorePath = path.join(root, ".gitignore");
+  const hasPackageJson = fs.existsSync(rootPackageJsonPath);
+  const isTurboExample =
+    repoInfo && repoInfo.username === "vercel" && repoInfo.name === "turbo";
+  const availableScripts = [];
+
+  // remove the meta file from turbo examples
+  if (isTurboExample || !repoInfo) {
+    try {
+      fs.rmSync(rootMetaJsonPath, { force: true });
+    } catch (err) {
+      //  ignore
+    }
+  }
+
+  if (hasPackageJson) {
+    let packageJsonContent;
+    try {
+      packageJsonContent = fs.readJsonSync(rootPackageJsonPath);
+    } catch {
+      // ignore
+    }
+
+    // if using the basic example, set the name to the project name (legacy behavior)
+    if (packageJsonContent) {
+      if (isDefaultExample) {
+        packageJsonContent.name = projectName;
+      }
+
+      // if we're using a pre-release version of create-turbo, install turbo canary instead of latest
+      const shouldUsePreRelease = semverPrerelease(cliPkgJson.version) !== null;
+      if (shouldUsePreRelease && packageJsonContent?.devDependencies?.turbo) {
+        packageJsonContent.devDependencies.turbo = "canary";
+      }
+
+      try {
+        fs.writeJsonSync(rootPackageJsonPath, packageJsonContent, {
+          spaces: 2,
+        });
+      } catch (err) {
+        // ignore
+      }
+
+      // read the scripts from the package.json
+      availableScripts.push(...Object.keys(packageJsonContent.scripts || {}));
+    }
+  }
+
+  // Copy `.gitignore` if the application did not provide one
+  addGitIgnore(ignorePath);
```

Comment:
hmm, why?  We should respect the template as-is, IMO.

Reply:
Missing `.turbo` in `.gitignore` is a huge pain point, I think it's very reasonable to include this if one is missing. `create-next-app` also does this.